### DISCUSSION TITLE
Sprite shake bug

### DISCUSSION THREAD

**Discussion by Only-a-cat** [ScratchAddons/ScratchAddons#1619] [Help] [answered]
On mine pc the resolution is 1366x768, and when i try to transfer a code / sound / costume to another sprite, it starts to shake. The problem is when i complete more than 2 lines of sprites, and the sprite bar shakes a lot, changing lines and I can't find what the sprite is sending to. If you can, plz solve this bug in this addon..

**Comment by cobaltt7** [comment]
same here!
________________________________

**Comment by TheColaber** [comment, marked as answer]
I have the same problem sometimes. However, this is not a problem from ScratchAddons. This is a scratch bug.

**Reply by WorldLanguages** [reply]
Original post never said it was our bug... read carefully

**Comment by cobaltt7** [comment]
@thecolaber we can still provide a fix for it...
________________________________

**Reply by TheColaber** [reply]
No, scratch should. You can make an issue if there isn't one already over there.

**Reply by cobaltt7** [reply]
It'd probably go in the bugs and glitches forum first

But we can give a fix, we gave one for the search load more glitch

**Reply by mxmou** [reply]
Fixing bugs isn't the purpose of Scratch Addons though. Scratch has many bugs and providing fixes for all of them wouldn't make sense for a browser extension intended to add new features.

**Reply by TheColaber** [reply]
Mind you, they have a GitHub (LLK) organization, you can go make an issue for that there.

**Reply by Hans5958** [reply]
> Fixing bugs isn't the purpose of Scratch Addons though. Scratch has many bugs and providing fixes for all of them wouldn't make sense for a browser extension intended to add new features.

Eh... I know some of you also publishes fixes on some parts of Scratch, such as the load more button, so what gives?

**Reply by mxmou** [reply]
> > Fixing bugs isn't the purpose of Scratch Addons though. Scratch has many bugs and providing fixes for all of them wouldn't make sense for a browser extension intended to add new features.
> 
> Eh... I know some of you also publishes fixes on some parts of Scratch, such as the load more button, so what gives?

Yes, I did that, but I think addons that fix Scratch bugs should be avoided anyway. Unless a generic "bug fix" addon is added, so they don't clutter the settings page.

**Reply by Hans5958** [reply]
> 
> 
> > > Fixing bugs isn't the purpose of Scratch Addons though. Scratch has many bugs and providing fixes for all of them wouldn't make sense for a browser extension intended to add new features.
> > 
> > 
> > Eh... I know some of you also publishes fixes on some parts of Scratch, such as the load more button, so what gives?
> 
> Yes, I did that, but I think addons that fix Scratch bugs should be avoided anyway. Unless a generic "bug fix" addon is added, so they don't clutter the settings page.

Eh, other than some heavy-resources-to-the-server addon ideas, I don't think we should not limit what goes in and what doesn't. It's a positive feature to fix bugs, and I don't think it is unethical to fix a broken part of a thing.

Also, about it being cluttered, I think we already have a cluttered settings page due to the amount of addons inside. This still can be improved, but for now I think it is not relevant at this time.

Is there any other reason that you don't want it? I think there's something that I don't know that could justify your reasoning.

**Reply by TheColaber** [reply]
Well, I'm sure if this could be fixed by an addon, someone could PR it to LLK like the previous bug that was fixed.
